File: src/sdk-reference/csharp/1/auth/get-my-credentials/snippets/get-my-credentials.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");
  string local_credentials = kuzzle.auth.getMyCredentials("local");

  Console.WriteLine(local_credentials);
  Console.WriteLine("Successfully got local credentials");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}

File: src/sdk-reference/cpp/1/auth/get-my-credentials/snippets/get-my-credentials.cpp
try {
  kuzzle->auth->login("local", R"({"username":"foo","password":"bar"})");
  std::string local_credentials = kuzzle->auth->getMyCredentials("local");

  std::cout << local_credentials << std::endl;
  std::cout << "Successfully got local credentials" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
